flowapi/flowapi/user_model.py

```diff
@@ -155,8 +161,8 @@ def can_get_results(self, *, query_json: dict) -> bool:
         UserClaimsVerificationError
             If the user cannot get the results of this kind of query at this level of aggregation
         """
-
-        return self.has_access(actions=["get_result"], query_json=query_json)
+        claims = await query_to_scopes(query_json)
```

Comment:
As above, but this time with link :) https://security.stackexchange.com/questions/145776/what-are-openid-scopes-and-claims

Reply:
Link copied to issue